www/include/configuration/configKnowledge/display-hosts.php

```diff
@@ -83,66 +71,100 @@
 $tpl = initSmartyTpl($modules_path, $tpl);
 
 try {
+    $postHost = !empty($_POST['searchHost'])
+        ? filter_input(INPUT_POST, 'searchHost', FILTER_SANITIZE_STRING)
+        : '';
+    $postHostgroup = !empty($_POST['searchHostgroup'])
+        ? filter_input(INPUT_POST, 'searchHostgroup', FILTER_VALIDATE_INT)
+        : false;
+    $postPoller = !empty($_POST['searchPoller'])
+        ? filter_input(INPUT_POST, 'searchPoller', FILTER_VALIDATE_INT)
+        : false;
+    $hasNoProcedure = !empty($_POST['searchHasNoProcedure'])
```

Comment:
Beware of the variable naming convention, the "has" prefix is traditionally used for the boolean type